.github/workflows/docs.yaml

```diff
@@ -66,7 +69,9 @@ jobs:
         run: git fetch --prune --tags --unshallow -f
       - name: Set output
         id: vars
-        run: echo "tag=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
+        run: |
+          echo "tag=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
+          echo "fdatetime=$(date +%Y%m%d-%H%M)" >> $GITHUB_OUTPUT
       - name: install anaconda-project
         run: conda install "anaconda-project>=0.10.1"
```

Comment:
The instructions in 00_Setup.ipynb has `anaconda-project>=0.11` but the docs.yaml uses `anaconda-project>=0.10.1`... is there a reason?

Reply:
Just that one was updated and not the other :)